features/manual_appointment_summary_data.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Manual appointment summary data
+  As a Pension Wise user with edit access
```

Comment:
I don't think the 'with edit access' should form part of the value proposition for this feature.